config/development.json

```diff
@@ -66,5 +66,8 @@
 	"NEST_LOG_LEVEL": "debug",
 	"SESSION_SECRET": "dev-session-secret",
 	"FEATURE_COURSE_SHARE": true,
-	"FEATURE_COLUMN_BOARD_ENABLED": true
+	"FEATURE_COLUMN_BOARD_ENABLED": true,
```

Comment:
we should not use the development.json. this is what's actually used in prod if it is not overwritten by the dof_app_deploy.

Reply:
oh sry dachte es war default.json

Reply:
Haha also kann es doch wieder rein? Ich habs jetzt rausgeworfen